content/docs/guides/schema-diff-tutorial.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Schema diff tutorial
+subtitle: Step-by-step guide showing you how to compare two development branches using Schema Diff
+enableTableOfContents: true
+---
+
+<ComingSoon/>
+
+In this guide we will create an initial schema on a new database called `staging` on our `main` branch. We'll then create two child development branches called `dev/alex` and `dev/jordan`, following our recommended convention for naming development branches. After making schema changes on `dev/jordan`, we we'll use the **Schema Diff** tool on the **Restore** page to get a side-by-side, Github-style visual comparison between these two development branches.
+
+## Step 1: Create the Initial Schema
+
+Start by creating a new `staging` database using the CLI. If you've got multiple projects, include `--project-id`.
+
+```bash
+neonctl databases create --name staging
+```
+
+Now create the initial schema that will serve as the base from which future changes and branches will diverge.
+
+1. First, copy your connection string:
+
+    ```bash
+    neonctl connection-string --database-name staging
+    ```
+
+2. Connect to the `staging`` with psql:
+
+    ```bash
+    psql 'postgres://neondb_owner:*********@ep-crimson-frost-a5i6p18z.us-east-2.aws.neon.tech/staging?sslmode=require'
+    ```
+
+2. Create the schema:
+
+    ```sql
+    CREATE TABLE person (
+        id SERIAL PRIMARY KEY,
+        name TEXT NOT NULL,
+        email TEXT UNIQUE NOT NULL
+    );
+    ```
+
+## Step 2: Create your team's branches
```

Comment:
To simply, how about changing this to "Create a development branch" and compare the schema from the dev branch to main?